src/containers/PageContainer/index.tsx

```diff
@@ -22,7 +22,6 @@ import wilderIcon from 'assets/WWLogo_SVG.svg';
 
```

Comment:
It looks like you did not rebase from `develop` branch. `PageContainer` component has some update with refactored `Header` component